cmd/kyma/dashboard/cmd.go

```diff
@@ -112,7 +112,8 @@ func (cmd *command) initContainerRunOpts(envs []string) (docker.ContainerRunOpts
 	kubeconfigPath := kube.KubeconfigPath(cmd.KubeconfigPath)
 	dashboardURL := fmt.Sprintf("http://localhost:%s", cmd.opts.Port)
 	if _, err := kube.NewFromConfig("", kubeconfigPath); err != nil {
-		cmd.CurrentStep.LogInfof("WARNING: kubeconfig is not mounted into container: %v", err)
+		cmd.CurrentStep.LogInfof("WARNING: There was a problem figuring out the current kubeconfig, "+
```

Comment:
```suggestion
		cmd.CurrentStep.LogInfof("WARNING: Could not identify the current kubeconfig. "+
```

Pls advise whether "identify" is the correct action. I assume this pops up if a) no kubeconfig was provided and b) if it's formatted in a way that we don't recognize automatically.

Reply:
yes, that's correct. I will apply your suggestion.